tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3243,6 +3299,31 @@ port::Status CudnnSupport::DoBatchNormalizationForwardImpl(
   float zero = 0.0;
   auto cudnn = cudnn_->GetHandle(parent_, stream);
 
+#if CUDNN_VERSION >= 7402
+  cudnnBatchNormOps_t bn_ops = CUDNN_BATCHNORM_OPS_BN;
+#endif
+  DeviceMemory<uint8> workspace;
+  DeviceMemory<uint8> reserve_space;
+  if (reserve_space_allocator != nullptr && workspace_allocator != nullptr) {
+#if CUDNN_VERSION >= 7402
```

Comment:
Move this outside the `if` and merge with the block that defines `bn_ops` above.